src/app/common/services/auth-service/AuthService.ts

```
@@ -108,31 +128,51 @@ export default class AuthService {
 
         const decodedToken = this.getDecodedAccessToken(token);
-        if (!decodedToken || !decodedToken.role) return false;
+        if (!decodedToken || !decodedToken.role) {
+            return false;
+        }
 
         return decodedToken.role === 'Admin';
     }
 
+    public static getUserRole(): UserRole | null {
+        const token = this.getAccessToken();
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)